demisto_sdk/commands/format/update_playbook.py

```diff
@@ -22,7 +22,19 @@ def __init__(self, input: str = '', output: str = '', path: str = '', from_versi
     def add_description(self):
         """Add empty description to playbook and tasks."""
         if self.verbose:
-            click.echo('Adding empty descriptions to relevant tasks')
+            click.echo('Adding descriptions for the playbook and to relevant tasks')
+        if 'description' not in set(self.data.keys()):
+            click.secho('No description is specified for this playbook, would you like to add a description? [Y/n]',
+                        fg='red')
+            user_answer = input()
+            if user_answer in ['n', 'N', 'no', 'No']:
+                print_error('Adding empty description for the playbook.')
+                user_description = ''
+            else:
+                click.secho('Please enter the description', fg='yellow')
+                user_description = input()
```

Comment:
```suggestion
                user_description = input("Please enter the description")
```

Don't think we need color here, this would do.

Also if a user simply clicks enter does it get an empty string or None? we should check that.

Reply:
If the user clicks enter, it will add the description key with an empty string.